manager/modelstore_interface/modelstore_interface.go

```diff
@@ -0,0 +1,13 @@
+package modelstore_interface
```

Comment:
I think we could name it as interface and name the package `modelstore_interface` to `modelstore`, WDYT

Reply:
+1 

`manager/modelstore_interface/modelstore_interface.go` -> `manager/modelstore/interface.go`